charts/spire/charts/spire-server/templates/pre-upgrade-hook.yaml

```diff
@@ -1,4 +1,9 @@
 {{- if eq ((dig "installAndUpgradeHooks" "enabled" .Values.controllerManager.installAndUpgradeHook.enabled .Values.global) | toString) "true" }}
+{{- $webhookname := printf "%s-%s-webhook" .Release.Namespace (include "spire-controller-manager.fullname" .) }}
+{{- $oldwebhookname := printf "%s-webhook" (include "spire-controller-manager.fullname" .) }}
+{{- if not (lookup "admissionregistration.k8s.io/v1" "ValidatingWebhookConfiguration" "" $webhookname) }}
+{{- $webhookname = $oldwebhookname }}
+{{- end }}
```

Comment:
These short term object renames need to be major releases. It's impossible to upgrade from 0.14.0 to 0.20.0 due to this rename logic being removed somewhere b/t here and 0.20.0